website/examples/text.js

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+import {Chart, Points, Text} from 'orama'
+
+const data = [
+  {date: new Date('2010-01-01'), value: 10, letter: 'A'},
+  {date: new Date('2010-02-01'), value: 17, letter: 'B'},
+  {date: new Date('2010-03-01'), value: 9, letter: 'C'},
+]
+
+export default (
+  <Chart>
+    <Points data={data} x="date" y="value" />
```

Comment:
It's a little difficult to see the text since it's rendering on top of the point. Could we either add `radiusValue={10}` to make the point bigger here (and thus see the text easier) or add `xOffsetValue={-10}` and change the `fillValue` of the `Text` so put the letter next to the point.

Reply:
I just copied all of these examples from the readme. Down to change both of them, but how about in another diff?

Reply:
Cool, that makes sense